src/lib/snack-bar/snack-bar-container.scss

```diff
@@ -12,12 +13,21 @@ $mat-snack-bar-max-width: 568px !default;
   border-radius: 2px;
   box-sizing: content-box;
   display: block;
+  margin: $mat-snack-bar-spacing-margin;
   max-width: $mat-snack-bar-max-width;
   min-width: $mat-snack-bar-min-width;
   padding: $mat-snack-bar-padding;
   // Initial transformation is applied to start snack bar out of view.
   transform: translateY(100%);
 
+  &.md-snack-bar-center {
```

Comment:
Add description comments for when these classes are applied / what they're used for